app/src/main/java/org/mifos/mobile/ui/registration/RegistrationVerificationFragment_Old.kt

```diff
@@ -18,13 +18,12 @@ import org.mifos.mobile.ui.login.LoginActivity
 import org.mifos.mobile.ui.fragments.base.BaseFragment
 import org.mifos.mobile.utils.RegistrationUiState
 import org.mifos.mobile.utils.Toaster
-import org.mifos.mobile.ui.registration.RegistrationViewModel
 
 /**
  * Created by dilpreet on 31/7/17.
  */
 @AndroidEntryPoint
-class RegistrationVerificationFragment : BaseFragment() {
+class RegistrationVerificationFragment_Old : BaseFragment() {
     private var _binding: FragmentRegistrationVerificationBinding? = null
```

Comment:
remove this fragment if not needed

Reply:
Rajan sir asked to add the old fragments in the meeting, that's why i am keeping them